2023-12-20 11:27:26 INFO main -- Started execution
2023-12-20 11:27:29 DEBUG Scraper -- Loading station codes
2023-12-20 11:27:29 DEBUG Scraper -- Station codes loaded
2023-12-20 11:27:29 INFO Scraper -- Scraping journeys on range: 2023-12-20 - 2023-12-20
2023-12-20 11:27:29 DEBUG Scraper -- Building query combos
2023-12-20 11:27:29 INFO Scraper -- 1 total connections
2023-12-20 11:27:29 DEBUG Scraper -- Scraping journeys on 2023-12-20
2023-12-20 11:27:29 DEBUG Scraper -- Initiated thread to scrape 100 connections
2023-12-20 11:27:34 DEBUG Scraper -- Scraped 2000000 -> 2004000 on 2023-12-20 in 5.031946699999935 seconds
2023-12-20 11:27:36 DEBUG Scraper -- Scraped 2000000 -> 2044000 on 2023-12-20 in 1.988643100000445 seconds
2023-12-20 11:27:39 DEBUG Scraper -- Scraped 2000000 -> 2030000 on 2023-12-20 in 2.3345196999998734 seconds
2023-12-20 11:27:42 DEBUG Scraper -- Scraped 2000000 -> 2060615 on 2023-12-20 in 3.6520055999999386 seconds
2023-12-20 11:27:45 DEBUG Scraper -- Scraped 2000000 -> 2060000 on 2023-12-20 in 2.6665530000000217 seconds
2023-12-20 11:27:47 DEBUG Scraper -- Scraped 2000000 -> 2040000 on 2023-12-20 in 2.2993453999997655 seconds
2023-12-20 11:27:49 DEBUG Scraper -- Scraped 2000000 -> 2038000 on 2023-12-20 in 1.8893041999999696 seconds
2023-12-20 11:27:51 DEBUG Scraper -- Scraped 2000000 -> 2024000 on 2023-12-20 in 2.0306232000002638 seconds
2023-12-20 11:27:54 DEBUG Scraper -- Scraped 2000000 -> 2024600 on 2023-12-20 in 2.3112275000003137 seconds
2023-12-20 11:27:56 DEBUG Scraper -- Scraped 2000000 -> 2064000 on 2023-12-20 in 2.6688363000002937 seconds
2023-12-20 11:27:59 DEBUG Scraper -- Scraped 2000000 -> 2044700 on 2023-12-20 in 2.3502464000002874 seconds
2023-12-20 11:28:01 DEBUG Scraper -- Scraped 2000000 -> 2064788 on 2023-12-20 in 1.9932111999996778 seconds
2023-12-20 11:28:03 DEBUG Scraper -- Scraped 2000000 -> 2014000 on 2023-12-20 in 2.4695887000007133 seconds
2023-12-20 11:28:05 DEBUG Scraper -- Scraped 2000000 -> 2030120 on 2023-12-20 in 2.3449870999993436 seconds
2023-12-20 11:28:08 DEBUG Scraper -- Scraped 2000000 -> 2020500 on 2023-12-20 in 2.410713900000701 seconds
2023-12-20 11:28:10 DEBUG Scraper -- Scraped 2000000 -> 2020000 on 2023-12-20 in 2.329201400000784 seconds
2023-12-20 11:28:12 DEBUG Scraper -- Scraped 2000000 -> 2030100 on 2023-12-20 in 2.0730416999995214 seconds
2023-12-20 11:28:14 DEBUG Scraper -- Scraped 2000000 -> 2024810 on 2023-12-20 in 1.8385526000001846 seconds
2023-12-20 11:28:16 DEBUG Scraper -- Scraped 2000000 -> 2044800 on 2023-12-20 in 2.2007452000007106 seconds
2023-12-20 11:28:18 DEBUG Scraper -- Scraped 2000000 -> 2060150 on 2023-12-20 in 1.9553785000007338 seconds
2023-12-20 11:28:20 DEBUG Scraper -- Scraped 2000000 -> 2064900 on 2023-12-20 in 2.1909172999994553 seconds
2023-12-20 11:28:23 DEBUG Scraper -- Scraped 2000000 -> 2034000 on 2023-12-20 in 2.3600996999994095 seconds
2023-12-20 11:28:25 DEBUG Scraper -- Scraped 2000000 -> 2024900 on 2023-12-20 in 1.9286715000007462 seconds
2023-12-20 11:28:27 DEBUG Scraper -- Scraped 2000000 -> 2054000 on 2023-12-20 in 2.3074735000000146 seconds
2023-12-20 11:28:29 DEBUG Scraper -- Scraped 2000000 -> 2034130 on 2023-12-20 in 2.314541600000666 seconds
2023-12-20 11:28:32 DEBUG Scraper -- Scraped 2000000 -> 2010000 on 2023-12-20 in 2.472367200000008 seconds
2023-12-20 11:28:43 WARNING QueryService MainThread -- TimeoutException for 2000000 -> 2028000
2023-12-20 11:28:43 ERROR QueryService MainThread -- Traceback (most recent call last):
  File "d:\Jannes\Documents\Trainspotting v2\src\miner\services\QueryService.py", line 54, in get
    WebDriverWait(self.driver, 10).until(EC.element_to_be_clickable((By.XPATH, '/html/body/div/div[2]/div[4]/div/div[1]/div/div[2]/div/form/div/div[2]/div/div[1]/div/span[2]'))).click()
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Jannes\Documents\Trainspotting v2\.conda\Lib\site-packages\selenium\webdriver\support\wait.py", line 80, in until
    raise TimeoutException(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: 


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "d:\Jannes\Documents\Trainspotting v2\src\miner\services\QueryService.py", line 65, in getQuery
    pageSource = self.get(url)
                 ^^^^^^^^^^^^^
  File "d:\Jannes\Documents\Trainspotting v2\src\miner\services\QueryService.py", line 56, in get
    self.handleException(None)
  File "d:\Jannes\Documents\Trainspotting v2\src\miner\services\QueryService.py", line 31, in handleException
    raise TimeoutException
selenium.common.exceptions.TimeoutException: Message: None


2023-12-20 11:28:43 DEBUG Scraper -- Scraped 2000000 -> 2028000 on 2023-12-20 in 11.079682700000376 seconds
2023-12-20 11:28:45 DEBUG Scraper -- Scraped 2000000 -> 2028156 on 2023-12-20 in 2.339944800000012 seconds
2023-12-20 11:28:47 DEBUG Scraper -- Scraped 2000000 -> 2024525 on 2023-12-20 in 1.9658741999992344 seconds
2023-12-20 11:28:58 WARNING QueryService MainThread -- TimeoutException for 2000000 -> 2064700
2023-12-20 11:28:58 ERROR QueryService MainThread -- Traceback (most recent call last):
  File "d:\Jannes\Documents\Trainspotting v2\src\miner\services\QueryService.py", line 54, in get
    WebDriverWait(self.driver, 10).until(EC.element_to_be_clickable((By.XPATH, '/html/body/div/div[2]/div[4]/div/div[1]/div/div[2]/div/form/div/div[2]/div/div[1]/div/span[2]'))).click()
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Jannes\Documents\Trainspotting v2\.conda\Lib\site-packages\selenium\webdriver\support\wait.py", line 80, in until
    raise TimeoutException(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: 


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "d:\Jannes\Documents\Trainspotting v2\src\miner\services\QueryService.py", line 65, in getQuery
    pageSource = self.get(url)
                 ^^^^^^^^^^^^^
  File "d:\Jannes\Documents\Trainspotting v2\src\miner\services\QueryService.py", line 56, in get
    self.handleException(None)
  File "d:\Jannes\Documents\Trainspotting v2\src\miner\services\QueryService.py", line 31, in handleException
    raise TimeoutException
selenium.common.exceptions.TimeoutException: Message: None


2023-12-20 11:28:58 DEBUG Scraper -- Scraped 2000000 -> 2064700 on 2023-12-20 in 10.822890299999926 seconds
2023-12-20 11:29:09 WARNING QueryService MainThread -- TimeoutException for 2000000 -> 2064700
2023-12-20 11:29:09 ERROR QueryService MainThread -- Traceback (most recent call last):
  File "d:\Jannes\Documents\Trainspotting v2\src\miner\services\QueryService.py", line 54, in get
    WebDriverWait(self.driver, 10).until(EC.element_to_be_clickable((By.XPATH, '/html/body/div/div[2]/div[4]/div/div[1]/div/div[2]/div/form/div/div[2]/div/div[1]/div/span[2]'))).click()
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Jannes\Documents\Trainspotting v2\.conda\Lib\site-packages\selenium\webdriver\support\wait.py", line 80, in until
    raise TimeoutException(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: 


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "d:\Jannes\Documents\Trainspotting v2\src\miner\services\QueryService.py", line 65, in getQuery
    pageSource = self.get(url)
                 ^^^^^^^^^^^^^
  File "d:\Jannes\Documents\Trainspotting v2\src\miner\services\QueryService.py", line 56, in get
    self.handleException(None)
  File "d:\Jannes\Documents\Trainspotting v2\src\miner\services\QueryService.py", line 31, in handleException
    raise TimeoutException
selenium.common.exceptions.TimeoutException: Message: None


2023-12-20 11:29:09 DEBUG Scraper -- Scraped 2000000 -> 2064700 on 2023-12-20 in 11.05945950000023 seconds
2023-12-20 11:29:11 DEBUG Scraper -- Scraped 2000000 -> 2040210 on 2023-12-20 in 1.9159436999998434 seconds
2023-12-20 11:29:22 WARNING QueryService MainThread -- TimeoutException for 2000000 -> 2028160
2023-12-20 11:29:22 ERROR QueryService MainThread -- Traceback (most recent call last):
  File "d:\Jannes\Documents\Trainspotting v2\src\miner\services\QueryService.py", line 54, in get
    WebDriverWait(self.driver, 10).until(EC.element_to_be_clickable((By.XPATH, '/html/body/div/div[2]/div[4]/div/div[1]/div/div[2]/div/form/div/div[2]/div/div[1]/div/span[2]'))).click()
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Jannes\Documents\Trainspotting v2\.conda\Lib\site-packages\selenium\webdriver\support\wait.py", line 80, in until
    raise TimeoutException(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: 


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "d:\Jannes\Documents\Trainspotting v2\src\miner\services\QueryService.py", line 65, in getQuery
    pageSource = self.get(url)
                 ^^^^^^^^^^^^^
  File "d:\Jannes\Documents\Trainspotting v2\src\miner\services\QueryService.py", line 56, in get
    self.handleException(None)
  File "d:\Jannes\Documents\Trainspotting v2\src\miner\services\QueryService.py", line 31, in handleException
    raise TimeoutException
selenium.common.exceptions.TimeoutException: Message: None


2023-12-20 11:29:22 DEBUG Scraper -- Scraped 2000000 -> 2028160 on 2023-12-20 in 11.189554999999928 seconds
2023-12-20 11:29:26 DEBUG Scraper -- Scraped 2000000 -> 2000125 on 2023-12-20 in 3.590453499999967 seconds
2023-12-20 11:29:27 DEBUG Scraper -- Scraped 2000000 -> 2001940 on 2023-12-20 in 1.5397893999997905 seconds
2023-12-20 11:29:30 DEBUG Scraper -- Scraped 2000000 -> 2024120 on 2023-12-20 in 2.714150700000573 seconds
2023-12-20 11:29:32 DEBUG Scraper -- Scraped 2000000 -> 2060620 on 2023-12-20 in 1.9250898999998753 seconds
2023-12-20 11:29:34 DEBUG Scraper -- Scraped 2000000 -> 2014130 on 2023-12-20 in 2.052495999999337 seconds
2023-12-20 11:29:36 DEBUG Scraper -- Scraped 2000000 -> 2058000 on 2023-12-20 in 1.9530711000006704 seconds
2023-12-20 11:29:38 DEBUG Scraper -- Scraped 2000000 -> 2020600 on 2023-12-20 in 1.8397611000000325 seconds
2023-12-20 11:29:40 DEBUG Scraper -- Scraped 2000000 -> 2000174 on 2023-12-20 in 2.4491985000004206 seconds
2023-12-20 11:29:43 DEBUG Scraper -- Scraped 2000000 -> 2060600 on 2023-12-20 in 2.3182441999997536 seconds
2023-12-20 11:29:45 DEBUG Scraper -- Scraped 2000000 -> 2064130 on 2023-12-20 in 1.9970210999999836 seconds
2023-12-20 11:29:47 DEBUG Scraper -- Scraped 2000000 -> 2000150 on 2023-12-20 in 2.443517900000188 seconds
2023-12-20 11:29:50 DEBUG Scraper -- Scraped 2000000 -> 2054555 on 2023-12-20 in 2.6650118999996266 seconds
2023-12-20 11:29:54 DEBUG Scraper -- Scraped 2000000 -> 2004600 on 2023-12-20 in 4.095931300000302 seconds
2023-12-20 11:30:05 WARNING QueryService MainThread -- TimeoutException for 2000000 -> 2040510
2023-12-20 11:30:05 ERROR QueryService MainThread -- Traceback (most recent call last):
  File "d:\Jannes\Documents\Trainspotting v2\src\miner\services\QueryService.py", line 54, in get
    WebDriverWait(self.driver, 10).until(EC.element_to_be_clickable((By.XPATH, '/html/body/div/div[2]/div[4]/div/div[1]/div/div[2]/div/form/div/div[2]/div/div[1]/div/span[2]'))).click()
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Jannes\Documents\Trainspotting v2\.conda\Lib\site-packages\selenium\webdriver\support\wait.py", line 80, in until
    raise TimeoutException(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: 


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "d:\Jannes\Documents\Trainspotting v2\src\miner\services\QueryService.py", line 65, in getQuery
    pageSource = self.get(url)
                 ^^^^^^^^^^^^^
  File "d:\Jannes\Documents\Trainspotting v2\src\miner\services\QueryService.py", line 56, in get
    self.handleException(None)
  File "d:\Jannes\Documents\Trainspotting v2\src\miner\services\QueryService.py", line 31, in handleException
    raise TimeoutException
selenium.common.exceptions.TimeoutException: Message: None


2023-12-20 11:30:05 DEBUG Scraper -- Scraped 2000000 -> 2040510 on 2023-12-20 in 11.023204699999951 seconds
2023-12-20 11:30:07 DEBUG Scraper -- Scraped 2000000 -> 2030600 on 2023-12-20 in 2.5534903000007034 seconds
2023-12-20 11:30:10 DEBUG Scraper -- Scraped 2000000 -> 2000427 on 2023-12-20 in 2.5052810999995927 seconds
2023-12-20 11:30:12 DEBUG Scraper -- Scraped 2000000 -> 2010050 on 2023-12-20 in 2.263542700000471 seconds
2023-12-20 11:30:14 DEBUG Scraper -- Scraped 2000000 -> 2048019 on 2023-12-20 in 1.8824869000000035 seconds
2023-12-20 11:30:17 DEBUG Scraper -- Scraped 2000000 -> 2060340 on 2023-12-20 in 2.737763299999642 seconds
2023-12-20 11:30:19 DEBUG Scraper -- Scraped 2000000 -> 2014370 on 2023-12-20 in 2.049894800000402 seconds
2023-12-20 11:30:30 WARNING QueryService MainThread -- TimeoutException for 2000000 -> 2030080
2023-12-20 11:30:30 ERROR QueryService MainThread -- Traceback (most recent call last):
  File "d:\Jannes\Documents\Trainspotting v2\src\miner\services\QueryService.py", line 54, in get
    WebDriverWait(self.driver, 10).until(EC.element_to_be_clickable((By.XPATH, '/html/body/div/div[2]/div[4]/div/div[1]/div/div[2]/div/form/div/div[2]/div/div[1]/div/span[2]'))).click()
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Jannes\Documents\Trainspotting v2\.conda\Lib\site-packages\selenium\webdriver\support\wait.py", line 80, in until
    raise TimeoutException(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: 


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "d:\Jannes\Documents\Trainspotting v2\src\miner\services\QueryService.py", line 65, in getQuery
    pageSource = self.get(url)
                 ^^^^^^^^^^^^^
  File "d:\Jannes\Documents\Trainspotting v2\src\miner\services\QueryService.py", line 56, in get
    self.handleException(None)
  File "d:\Jannes\Documents\Trainspotting v2\src\miner\services\QueryService.py", line 31, in handleException
    raise TimeoutException
selenium.common.exceptions.TimeoutException: Message: None


2023-12-20 11:30:30 DEBUG Scraper -- Scraped 2000000 -> 2030080 on 2023-12-20 in 10.904433899999276 seconds
2023-12-20 11:30:32 DEBUG Scraper -- Scraped 2000000 -> 2000351 on 2023-12-20 in 2.4153784999998607 seconds
2023-12-20 11:30:34 DEBUG Scraper -- Scraped 2000000 -> 2050000 on 2023-12-20 in 2.150407900000573 seconds
2023-12-20 11:30:37 DEBUG Scraper -- Scraped 2000000 -> 2064180 on 2023-12-20 in 2.4000451000001704 seconds
2023-12-20 11:30:48 WARNING QueryService MainThread -- TimeoutException for 2000000 -> 2020832
2023-12-20 11:30:48 ERROR QueryService MainThread -- Traceback (most recent call last):
  File "d:\Jannes\Documents\Trainspotting v2\src\miner\services\QueryService.py", line 54, in get
    WebDriverWait(self.driver, 10).until(EC.element_to_be_clickable((By.XPATH, '/html/body/div/div[2]/div[4]/div/div[1]/div/div[2]/div/form/div/div[2]/div/div[1]/div/span[2]'))).click()
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Jannes\Documents\Trainspotting v2\.conda\Lib\site-packages\selenium\webdriver\support\wait.py", line 80, in until
    raise TimeoutException(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: 


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "d:\Jannes\Documents\Trainspotting v2\src\miner\services\QueryService.py", line 65, in getQuery
    pageSource = self.get(url)
                 ^^^^^^^^^^^^^
  File "d:\Jannes\Documents\Trainspotting v2\src\miner\services\QueryService.py", line 56, in get
    self.handleException(None)
  File "d:\Jannes\Documents\Trainspotting v2\src\miner\services\QueryService.py", line 31, in handleException
    raise TimeoutException
selenium.common.exceptions.TimeoutException: Message: None


2023-12-20 11:30:48 DEBUG Scraper -- Scraped 2000000 -> 2020832 on 2023-12-20 in 11.287373600000137 seconds
2023-12-20 11:30:50 DEBUG Scraper -- Scraped 2000000 -> 2000170 on 2023-12-20 in 2.110506600000008 seconds
2023-12-20 11:30:53 DEBUG Scraper -- Scraped 2000000 -> 2000065 on 2023-12-20 in 3.3802802000000156 seconds
2023-12-20 11:30:56 DEBUG Scraper -- Scraped 2000000 -> 2024700 on 2023-12-20 in 2.2070653999999195 seconds
2023-12-20 11:30:58 DEBUG Scraper -- Scraped 2000000 -> 2010041 on 2023-12-20 in 2.562326899999789 seconds
2023-12-20 11:31:09 WARNING QueryService MainThread -- TimeoutException for 2000000 -> 2040600
2023-12-20 11:31:09 ERROR QueryService MainThread -- Traceback (most recent call last):
  File "d:\Jannes\Documents\Trainspotting v2\src\miner\services\QueryService.py", line 54, in get
    WebDriverWait(self.driver, 10).until(EC.element_to_be_clickable((By.XPATH, '/html/body/div/div[2]/div[4]/div/div[1]/div/div[2]/div/form/div/div[2]/div/div[1]/div/span[2]'))).click()
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Jannes\Documents\Trainspotting v2\.conda\Lib\site-packages\selenium\webdriver\support\wait.py", line 80, in until
    raise TimeoutException(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: 


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "d:\Jannes\Documents\Trainspotting v2\src\miner\services\QueryService.py", line 65, in getQuery
    pageSource = self.get(url)
                 ^^^^^^^^^^^^^
  File "d:\Jannes\Documents\Trainspotting v2\src\miner\services\QueryService.py", line 56, in get
    self.handleException(None)
  File "d:\Jannes\Documents\Trainspotting v2\src\miner\services\QueryService.py", line 31, in handleException
    raise TimeoutException
selenium.common.exceptions.TimeoutException: Message: None


2023-12-20 11:31:09 DEBUG Scraper -- Scraped 2000000 -> 2040600 on 2023-12-20 in 11.14489599999979 seconds
2023-12-20 11:31:12 DEBUG Scraper -- Scraped 2000000 -> 2010290 on 2023-12-20 in 2.477865599999859 seconds
2023-12-20 11:31:14 DEBUG Scraper -- Scraped 2000000 -> 2000140 on 2023-12-20 in 2.57131439999921 seconds
2023-12-20 11:31:17 DEBUG Scraper -- Scraped 2000000 -> 2010400 on 2023-12-20 in 2.2879824000001463 seconds
2023-12-20 11:31:19 DEBUG Scraper -- Scraped 2000000 -> 2064070 on 2023-12-20 in 2.1299552000000403 seconds
2023-12-20 11:31:21 DEBUG Scraper -- Scraped 2000000 -> 2030308 on 2023-12-20 in 2.213894999999866 seconds
2023-12-20 11:31:23 DEBUG Scraper -- Scraped 2000000 -> 2060593 on 2023-12-20 in 1.8935111999999208 seconds
2023-12-20 11:31:34 WARNING QueryService MainThread -- TimeoutException for 2000000 -> 2024780
2023-12-20 11:31:34 ERROR QueryService MainThread -- Traceback (most recent call last):
  File "d:\Jannes\Documents\Trainspotting v2\src\miner\services\QueryService.py", line 54, in get
    WebDriverWait(self.driver, 10).until(EC.element_to_be_clickable((By.XPATH, '/html/body/div/div[2]/div[4]/div/div[1]/div/div[2]/div/form/div/div[2]/div/div[1]/div/span[2]'))).click()
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Jannes\Documents\Trainspotting v2\.conda\Lib\site-packages\selenium\webdriver\support\wait.py", line 80, in until
    raise TimeoutException(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: 


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "d:\Jannes\Documents\Trainspotting v2\src\miner\services\QueryService.py", line 65, in getQuery
    pageSource = self.get(url)
                 ^^^^^^^^^^^^^
  File "d:\Jannes\Documents\Trainspotting v2\src\miner\services\QueryService.py", line 56, in get
    self.handleException(None)
  File "d:\Jannes\Documents\Trainspotting v2\src\miner\services\QueryService.py", line 31, in handleException
    raise TimeoutException
selenium.common.exceptions.TimeoutException: Message: None


2023-12-20 11:31:34 DEBUG Scraper -- Scraped 2000000 -> 2024780 on 2023-12-20 in 11.270343700000012 seconds
2023-12-20 11:31:37 DEBUG Scraper -- Scraped 2000000 -> 2004200 on 2023-12-20 in 2.701701199999661 seconds
2023-12-20 11:31:39 DEBUG Scraper -- Scraped 2000000 -> 2010090 on 2023-12-20 in 1.9873703999992358 seconds
2023-12-20 11:31:41 DEBUG Scraper -- Scraped 2000000 -> 2064110 on 2023-12-20 in 2.156756899999891 seconds
2023-12-20 11:31:44 DEBUG Scraper -- Scraped 2000000 -> 2014700 on 2023-12-20 in 2.6442736000008153 seconds
2023-12-20 11:31:46 DEBUG Scraper -- Scraped 2000000 -> 2005000 on 2023-12-20 in 2.1338753999998517 seconds
2023-12-20 11:31:49 DEBUG Scraper -- Scraped 2000000 -> 2000015 on 2023-12-20 in 2.7541543999996065 seconds
2023-12-20 11:31:51 DEBUG Scraper -- Scraped 2000000 -> 2064055 on 2023-12-20 in 2.1032445999999254 seconds
2023-12-20 11:31:53 DEBUG Scraper -- Scraped 2000000 -> 2064287 on 2023-12-20 in 1.948364799999581 seconds
2023-12-20 11:32:04 WARNING QueryService MainThread -- TimeoutException for 2000000 -> 2024793
2023-12-20 11:32:04 ERROR QueryService MainThread -- Traceback (most recent call last):
  File "d:\Jannes\Documents\Trainspotting v2\src\miner\services\QueryService.py", line 54, in get
    WebDriverWait(self.driver, 10).until(EC.element_to_be_clickable((By.XPATH, '/html/body/div/div[2]/div[4]/div/div[1]/div/div[2]/div/form/div/div[2]/div/div[1]/div/span[2]'))).click()
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Jannes\Documents\Trainspotting v2\.conda\Lib\site-packages\selenium\webdriver\support\wait.py", line 80, in until
    raise TimeoutException(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: 


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "d:\Jannes\Documents\Trainspotting v2\src\miner\services\QueryService.py", line 65, in getQuery
    pageSource = self.get(url)
                 ^^^^^^^^^^^^^
  File "d:\Jannes\Documents\Trainspotting v2\src\miner\services\QueryService.py", line 56, in get
    self.handleException(None)
  File "d:\Jannes\Documents\Trainspotting v2\src\miner\services\QueryService.py", line 31, in handleException
    raise TimeoutException
selenium.common.exceptions.TimeoutException: Message: None


2023-12-20 11:32:04 DEBUG Scraper -- Scraped 2000000 -> 2024793 on 2023-12-20 in 10.87583799999993 seconds
2023-12-20 11:32:05 DEBUG Scraper -- Scraped 2000000 -> 2000029 on 2023-12-20 in 1.2691113000000769 seconds
2023-12-20 11:32:16 WARNING QueryService MainThread -- TimeoutException for 2000000 -> 2034630
2023-12-20 11:32:16 ERROR QueryService MainThread -- Traceback (most recent call last):
  File "d:\Jannes\Documents\Trainspotting v2\src\miner\services\QueryService.py", line 54, in get
    WebDriverWait(self.driver, 10).until(EC.element_to_be_clickable((By.XPATH, '/html/body/div/div[2]/div[4]/div/div[1]/div/div[2]/div/form/div/div[2]/div/div[1]/div/span[2]'))).click()
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Jannes\Documents\Trainspotting v2\.conda\Lib\site-packages\selenium\webdriver\support\wait.py", line 80, in until
    raise TimeoutException(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: 


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "d:\Jannes\Documents\Trainspotting v2\src\miner\services\QueryService.py", line 65, in getQuery
    pageSource = self.get(url)
                 ^^^^^^^^^^^^^
  File "d:\Jannes\Documents\Trainspotting v2\src\miner\services\QueryService.py", line 56, in get
    self.handleException(None)
  File "d:\Jannes\Documents\Trainspotting v2\src\miner\services\QueryService.py", line 31, in handleException
    raise TimeoutException
selenium.common.exceptions.TimeoutException: Message: None


2023-12-20 11:32:16 DEBUG Scraper -- Scraped 2000000 -> 2034630 on 2023-12-20 in 10.957167599999593 seconds
2023-12-20 11:32:18 DEBUG Scraper -- Scraped 2000000 -> 2005594 on 2023-12-20 in 2.2518730000001597 seconds
2023-12-20 11:32:20 DEBUG Scraper -- Scraped 2000000 -> 2002080 on 2023-12-20 in 1.7449161000004096 seconds
2023-12-20 11:32:22 DEBUG Scraper -- Scraped 2000000 -> 2064201 on 2023-12-20 in 2.1058503999993263 seconds
2023-12-20 11:32:24 DEBUG Scraper -- Scraped 2000000 -> 2005601 on 2023-12-20 in 1.922694699999738 seconds
2023-12-20 11:32:26 DEBUG Scraper -- Scraped 2000000 -> 2054775 on 2023-12-20 in 1.9484184000002642 seconds
2023-12-20 11:32:28 DEBUG Scraper -- Scraped 2000000 -> 2014340 on 2023-12-20 in 2.1188851999995677 seconds
2023-12-20 11:32:30 DEBUG Scraper -- Scraped 2000000 -> 2054260 on 2023-12-20 in 2.139927700000044 seconds
2023-12-20 11:32:32 DEBUG Scraper -- Scraped 2000000 -> 2010280 on 2023-12-20 in 2.0895126000004893 seconds
2023-12-20 11:32:35 DEBUG Scraper -- Scraped 2000000 -> 2060420 on 2023-12-20 in 2.4698722000002817 seconds
2023-12-20 11:32:37 DEBUG Scraper -- Scraped 2000000 -> 2004500 on 2023-12-20 in 2.1703158000000258 seconds
2023-12-20 11:32:39 DEBUG Scraper -- Scraped 2000000 -> 2040480 on 2023-12-20 in 1.885347599999477 seconds
2023-12-20 11:32:41 DEBUG Scraper -- Scraped 2000000 -> 2000480 on 2023-12-20 in 2.492224399999941 seconds
2023-12-20 11:32:43 DEBUG Scraper -- Scraped 2000000 -> 2064291 on 2023-12-20 in 2.125193799999579 seconds
2023-12-20 11:32:45 DEBUG Scraper -- Scraped 2000000 -> 2038230 on 2023-12-20 in 1.9583034999996016 seconds
2023-12-20 11:32:47 DEBUG Scraper -- Scraped 2000000 -> 2020865 on 2023-12-20 in 2.1162248000000545 seconds
2023-12-20 11:32:49 DEBUG Scraper -- Scraped 2000000 -> 2044720 on 2023-12-20 in 1.9004672999999457 seconds
2023-12-20 11:32:53 DEBUG Scraper -- Scraped 2000000 -> 2000055 on 2023-12-20 in 3.75836419999996 seconds
2023-12-20 11:32:55 DEBUG Scraper -- Scraped 2000000 -> 2034750 on 2023-12-20 in 1.9346473999994487 seconds
2023-12-20 11:33:06 WARNING QueryService MainThread -- TimeoutException for 2000000 -> 2028093
2023-12-20 11:33:06 ERROR QueryService MainThread -- Traceback (most recent call last):
  File "d:\Jannes\Documents\Trainspotting v2\src\miner\services\QueryService.py", line 54, in get
    WebDriverWait(self.driver, 10).until(EC.element_to_be_clickable((By.XPATH, '/html/body/div/div[2]/div[4]/div/div[1]/div/div[2]/div/form/div/div[2]/div/div[1]/div/span[2]'))).click()
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Jannes\Documents\Trainspotting v2\.conda\Lib\site-packages\selenium\webdriver\support\wait.py", line 80, in until
    raise TimeoutException(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: 


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "d:\Jannes\Documents\Trainspotting v2\src\miner\services\QueryService.py", line 65, in getQuery
    pageSource = self.get(url)
                 ^^^^^^^^^^^^^
  File "d:\Jannes\Documents\Trainspotting v2\src\miner\services\QueryService.py", line 56, in get
    self.handleException(None)
  File "d:\Jannes\Documents\Trainspotting v2\src\miner\services\QueryService.py", line 31, in handleException
    raise TimeoutException
selenium.common.exceptions.TimeoutException: Message: None


2023-12-20 11:33:06 DEBUG Scraper -- Scraped 2000000 -> 2028093 on 2023-12-20 in 11.30639500000052 seconds
2023-12-20 11:33:06 INFO Scraper -- Writing to journeys_stats_20_12_2023.parquet
2023-12-20 11:33:07 INFO Scraper -- Finished writing to journeys_stats_20_12_2023.parquet in 0.7737906000002113
2023-12-20 11:33:07 DEBUG Scraper -- Completed scraping journeys on 2023-12-20 in 337.6325308000005
2023-12-20 11:33:30 INFO Scraper -- Finished scraping!
2023-12-20 11:33:30 INFO main -- Finished execution in 364.5103760000002 seconds
